solutions.md

```diff
@@ -0,0 +1,9 @@
+solutions.md
+
+https://jsfiddle.net/dave_sloan/Lkqup3eg/17/
+
+Hm, I really struggled with this assignment.  I think I need to go back and do some basic CRUD / routing work.  Maybe the original Super-Crud assignment.  I think what I need is a step by step tutorial to go through basic building of CRUD routes.  Maybe for Spotify or for Books.   Or Wine.  
+
```

Comment:
@slnwlf thanks for the note! This was definitely a challenging assignment that brought a lot of different pieces together. Good idea to go over the original Super CRUD to make comparisons to the way we do things in Angular. We will also do more practice today and during review on Friday. Your weekend lab will also be a good time to practice the basics if you keep it simple.